docs/docs/cli/generated/tracetest_completion_bash.md

```diff
@@ -0,0 +1,54 @@
+# CLI Referece
+## tracetest completion bash
+
+Generate the autocompletion script for bash
+
+### Synopsis
+
+Generate the autocompletion script for the bash shell.
+
+This script depends on the 'bash-completion' package.
```

Comment:
nit: Is there a way to put a link to the `bash-completion` package?

Reply:
This is autogenerated, I don't think it's possible to change this specific command